CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # v3.8.0
 
 ### Added
+
```

Comment:
I think those changes can be reverted as this file is autogenerated.

Reply:
Since `prettier` is being included, I assumed you would want it run, even though there is no script for it. Running `prettier` on the project by default will perform this change (and that below). (It also changes some YML in `.github`, though I didn't include those, not knowing if they were safe.) I could include a `.prettierignore` to avoid changing these files, however.

Reply:
I'd say autogenerated files should be ignored

Reply:
Ok, I've added an ignore file, reverted the changes to the two Markdown files, and added two prettier scripts to package.json.